app/models/salt_handler/minion_start.rb

```diff
@@ -25,6 +25,9 @@ def process_event
     return false if ["ca", "admin"].include? id
 
     minion_info = Velum::Salt.minions[id]
+    puts Velum::Salt.minions
+    puts id
+    puts minion_info
```

Comment:
looks like this is only debug output? in general if you want to have logs appear in the velum container logs, just use e.g. `Rails.logger.debug(msg)`

Reply:
Missed to delete this.